### ISSUE TITLE
"How is this UI element made?" context menu action

### ISSUE THREAD

**Issue by aartaka** [atlas-engineer/nyxt#2860]
**Is your feature request related to a problem? Please describe.**
Our interfaces are mostly `internal-page`s with Spinneret tags in them. Those are quite opaque, especially when compiled to HTML. It would be nice to have a view (in several levels of magnification) of what the page is made of.

**Describe the solution you'd like**
I'm thinking of `:ncontext` tag that would preserve the Spinneret tags passed into it, untact as a string attribute somewhere. And internal page HTML also preserving the name of the internal page and its args. Then, a single command could collect all these breadcrumbs and render then in an approachable way.

**Describe alternatives you've considered**
Maybe we should've used [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) if our interfaces from the beginning?

**Additional context**
@Ambrevar had this idea from Pharo, and I had this idea in parallel with him from the Spinneret tags overdose.
